vadl/main/vadl/gcb/passes/relocation/model/AutomaticallyGeneratedRelocation.java

```diff
@@ -170,4 +170,9 @@ public Format.Field field() {
   public Format format() {
     return format;
   }
+
+  @Override
+  public String llvmKind() {
+    return kind.llvmKind();
+  }
```

Comment:
minor: just add a method to return the kind and all the users can call llvmKind themselves.